resources/views/Checkout/OrderConfirmation.twig

```diff
@@ -28,6 +29,11 @@
             {% include "Ceres::Checkout.OrderDetails" with { orderData: data } %}
 
             <hr>
+            {% if addContentAfterOrderDetailsRow %}
```

Comment:
könntest du den Check ob content in dem Container liegt bitte anpassen:

{% if addContentAfterOrderDetailsRow |trim is not empty %}